chat-plugins/modlog.js

```diff
@@ -145,12 +137,70 @@ function getMoreButton(room, search, useExactSearch, lines, maxLines) {
 	if (!newLines || lines < maxLines) {
 		return ''; // don't show a button if no more pre-set increments are valid or if the amount of results is already below the max
 	} else {
-		if (useExactSearch) search = Chat.escapeHTML(`"${search}"`);
+		search = Chat.escapeHTML(`${search}`);
```

Comment:
Shouldn't ``search`` be surrounded by quotes?

Reply:
As I understand it, the quotes will be left in the search term and instead handled in the child process in this impementation, meaning there's no need to re-add them here.

Reply:
Can you confirm @sparkychildcharlie ?

Reply:
yep i can! I'm just unable to add reactions on mobile github :(